sentry_sdk/hub.py

```diff
@@ -464,6 +465,12 @@ def start_span(
         for every incoming HTTP request. Use `start_transaction` to start a new
         transaction when one is not already in progress.
         """
+        instrumenter = kwargs.get("instrumenter", INSTRUMENTER.SENTRY)
```

Comment:
I would prefer an explicit arg `instrumenter=INSTRUMENTER.SENTRY` if you remove the redundant check in tracing.py, then this doesn't need to passed down to the constructor.

Reply:
Main reason for having it in `kwargs` was that I did not want to change the function definition to not break user code eventually. But after thinking a bit I guess when setting a default in the function parameter there is little chance to break user code